_posts/2017-03-14-pycon-pune-2017-a-wonderful-python-experience.md

```diff
@@ -0,0 +1,220 @@
+
+I am sad to declare that I was unable to attend any keynote speeches because of
+the illness. Mostly I rested at the hotel or talked with people during the
+conference days.
+
+
+![pycon_pune_2017_keynote_speackers]({{
+site.url}}/assets/images/pycon_pune_2017_keynote_speackers.jpg)
+
+
+#### Volunteer Party
+
+If you are volunteering for this conference, then you will be invited to a
+volunteer dinner party. We enjoyed party colored disco lights dancing on the
+bits of the DJ. The Punjabi food and if you are above 25 than you were allowed
+to take a sip of a beer.
+
+#### Devsprint
+
+Devsprint happen at the [Redhat Headquarters,
+Pune](https://goo.gl/maps/mXeirzQhPFz). I found the building has tight security.
+You will find an individual pantry section dedicated to each department. We were
+instructed to hack at huge cafeteria section. I myself contributed to Core
+Python. Nick Coghlan was mentoring for Core Python. I managed to review one PR,
+find one broken test case and written fix of an existing issue with the help of
+him. Honza was leading the development of Django web framework. A team of [Anand
+Chitipothu](http://anandology.com/) mentored for [Web2py](www.web2py.com/).
+[Farhaan Bukhsh](https://twitter.com/fhackdroid) mentored for
+[Pagure](https://github.com/pypingou/pagure). John Hawley encouraged on
+contributing to [MicroPython](https://micropython.org/). Terr Oda, Stephen
+Turnbull and Florian Fuchs mentored for
+[GNU/Mailman](https://en.wikipedia.org/wiki/GNU_Mailman).
+
+
+![cpython_devsprint]({{
+site.url}}/assets/images/cpython_devsprint.jpg)
+
+#### Why to attend this conference?
+
+* This conference has coolest Devsprint. The organizers understand the value of
+  the Devsprint in a conference. I have never observed such importance of
+  Devsprint at any other Python conference happening in India.
+
+* If you are a student, then this is the beginner friendly conference. You
+  should not be a worry to attend if you are a Python noobs. You will receive a
+  student concession for the tickets too.
+
+* If you are a developer, coming to this conference will inspire you to grow
+  from your present level. You will meet core contributors, lead programmers,
+  owners of startup, project managers. You will find the huge scope of
+  networking with people.
+
+* The conference is single track event. This decision was helped me to not loose
+  the interesting talks. In my previous experience, parallel tracks killed me to
+  leave one and attend other despite I am interested in attending both.
+
+* I have never seen such huge amount of keynote speakers at any conference
+  happening in India. Keynote speakers were the main attraction of this
+  conference.
+
+#### What was missing?
+
+* If you are a swag-hungry fellow than attending this conference will not worth
+  to you. The conference attendees has to be satisfied with the conference
+  T-shirt.
+
+* I observed the number of corporate stalls was less than compared to other
```

Comment:
This reads awkwardly - I would say "I observed there were fewer corporate stalls than at other Python conferences".